fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -189,4 +190,87 @@ object FAPI {
     val backfillRequest = BackfillResolver.resolveFromConfig(collectionConfig)
     BackfillResolver.backfill(backfillRequest, adjustSearchQuery, adjustItemQuery)
   }
+
+  private def updateContentProperties(content: FaciaContent, properties: ContentProperties): FaciaContent = content match {
+    case curatedContent: CuratedContent => curatedContent.copy(properties = properties)
+    case supportingCuratedContent: SupportingCuratedContent => supportingCuratedContent.copy(properties = properties)
+    case snap: LatestSnap => snap.copy(properties = properties)
+    case snap: LinkSnap => snap.copy(properties = properties)
+  }
+
+  private def updateGroup(content: FaciaContent, group: String): FaciaContent = content match {
+    case curatedContent: CuratedContent => curatedContent.copy(group = group)
+    case supportingCuratedContent: SupportingCuratedContent => supportingCuratedContent.copy(group = group)
+    case snap: LatestSnap => snap.copy(group = group)
+    case snap: LinkSnap => snap.copy(group = group)
+  }
+
+  private def applyDefaultBoost[T](
+    content: T,
+    default: BoostLevel,
+    allowedBoosts: List[BoostLevel],
+    getBoostLevel: T => BoostLevel,
+    setBoostLevel: (T, BoostLevel) => T
+  ): T =
+    if (allowedBoosts.contains(getBoostLevel(content)))
+      content
+    else
+      setBoostLevel(content, default)
+
+  case class GroupBoostConfig(default: BoostLevel, allowedBoosts: List[BoostLevel], maxItems: Int, group: String)
+
+  private val flexibleGeneralBoosts = List(
```

Comment:
I think I remember these were the agreed defaults for backfilled content, but just wanted to double check - so 1 item in the splash, nothing in the very big or big groups, and 8 stories in standard?

Reply:
These match up with https://github.com/guardian/facia-tool/blob/33614ebfe9d2ccda4d4dc70e7a8c0a9f4dce1b70/public/src/js/constants/defaults.js#L126 but in practice the values here will never actually be used, since the config will always include maxItems which override these defaults

Reply:
Ah yeah, of course!